src/native/src/components/Error/index.tsx

```diff
@@ -1,4 +1,4 @@
-import Text from "../Text";
+import Text from "../BaseText";
```

Comment:
Acho q pra manter o padraozinho, já q tu trocou o nome do componente principal, ou tu prefere trocar isso outro momento?

```suggestion
import BaseText from "../BaseText";
```